blob/fileblob/fileblob.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // Package fileblob provides a blob implementation that uses the filesystem.
-// Use OpenBucket to construct a blob.Bucket.
+// Use OpenBucket to construct a *blob.Bucket.
```

Comment:
This doesn't make sense. The thing being constructed is a blob.Bucket. "Constructing" the pointer is trivial.

Reply:
I'm updating this for consistency with other package docstrings. It feels more accurate because that's what the function is actually returning, no?

Reply:
Okay, I guess no one will really be confused by it.